editor/components/page-attributes/order.js

```diff
@@ -27,20 +26,17 @@ export function PageAttributes( { onUpdateOrder, instanceId, order } ) {
 	// Create unique identifier for inputs
 	const inputId = `editor-page-attributes__order-${ instanceId }`;
 
-	return (
-		<PageAttributesCheck>
-			<label htmlFor={ inputId }>
-				{ __( 'Order' ) }
-			</label>
-			<input
-				type="text"
-				value={ order || 0 }
-				onChange={ setUpdatedOrder }
-				id={ inputId }
-				size={ 6 }
-			/>
-		</PageAttributesCheck>
-	);
+	return ( <div>
```

Comment:
Can I ask why did you remove the `PageAttributesCheck` here? We should avoid rendering this component if the post type doesn't support it. Since this component is reusable, it should be fine to duplicate this check in here.